### DISCUSSION TITLE
npmjs discord community

### DISCUSSION THREAD

**Discussion by subnwa** [npm/feedback#354] [General]
hello i want to create a discord community for npm js do i need legal permission or a different permission for this

**Comment by MylesBorins** [comment]
Hey @penguenn. At this time we are not going to bless an official npmjs discord community. You are more than welcome to make your own community on discord but if you want to refer to npmjs in any capacity you will need to make it clear that it is not an official discord or in anyway associated with npmjs or GitHub.

**Reply by subnwa** [reply]
ok thanks a lot, would it be a problem if I share the invitation of the community here

**Reply by MylesBorins** [reply]
@penguenn this feedback repository is for gathering feedback, not for marketing or community outreach. It would be inappropriate to share it here.

In general if I were to help promote an online community I would want to know the people running it, how it is being moderated, the value it is bringing, and why folks should go there there instead of a variety of existing community locations (such as Freenode IRC and OpenJS Slack).

I wish you luck with starting a new community but do want to be explicit that you cannot use the name npmjs to promote it nor can you infer in any way that the npmjs team or GitHub endorse your community.

**Reply by subnwa** [reply]
Is it ok to have npm on the server name?

Server Name: **npm Hub**

**Reply by MylesBorins** [reply]
@penguenn please refer to our [trademark policy](https://www.npmjs.com/policies/trademark). My primary ask is you make it clear that the community is not officially endorsed or supported by npmjs or GitHub
